views/web.py

```diff
@@ -197,9 +203,12 @@ def leaderboard():
         top_lovers=top_lover_dicts,
         departments=logic.department.META_DEPARTMENTS,
         sub_departments=logic.department.META_DEPARTMENT_MAP,
+        offices=logic.office.OFFICES,
         selected_dept=department,
-        selected_timespan=timespan,
+        selected_timespan=timespan, selected_office=office,
         org_title=config.ORG_TITLE,
```

Comment:
I wonder if org_title is still being used? If not it should be removed here and in the config.